sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -509,7 +509,7 @@ class Analyzer(override val catalogManager: CatalogManager)
     }
 
     private def hasUnresolvedAlias(exprs: Seq[NamedExpression]) =
-      exprs.exists(_.find(_.isInstanceOf[UnresolvedAlias]).isDefined)
+      exprs.exists(_.exists(_.isInstanceOf[UnresolvedAlias]))
```

Comment:
@zhengruifeng In addition, if `f` becomes `PartialFunction`, there may need to be rewritten as
```
exprs.exists(_.exists { case _: UnresolvedAlias => true})
```

Reply:
yes, If we apply this PartialFunction, call sites like this need to be rewritten.

Reply:
OK

Reply:
Let's wait for more suggestions from others